addons/hr_fleet/security/ir.model.access.csv

```diff
@@ -1,2 +1,13 @@
 id,name,model_id/id,group_id/id,perm_read,perm_write,perm_create,perm_unlink
 hr_fleet_vehicle_access_right_hr_officer,hr_fleet_vehicle_access_right_hr_officer,model_fleet_vehicle,hr.group_hr_user,1,0,0,0
+hr_fleet_vehicle_log_services_access_right_user,fleet_vehicle_log_services_access_right,model_fleet_vehicle_log_services,base.group_user,1,0,0,0
+hr_fleet_vehicle_log_contract_access_right_user,fleet_vehicle_log_contract_access_right,model_fleet_vehicle_log_contract,base.group_user,1,0,0,0
+hr_fleet_vehicle_assignation_log_base_group_user,fleet_vehicle_assignation_log,model_fleet_vehicle_assignation_log,base.group_user,1,0,0,0
+hr_fleet_vehicle_odometer_access_right,fleet_vehicle_odometer_access_right,model_fleet_vehicle_odometer,base.group_user,1,0,0,0
+hr_fleet_service_type_access_right_user,fleet_service_type_access_right,fleet.model_fleet_service_type,base.group_user,1,0,0,0
+hr_fleet_vehicle_model_access_right_user,fleet_vehicle_model_access_right,fleet.model_fleet_vehicle_model,base.group_user,1,0,0,0
+hr_fleet_vehicle_tag_access_right_user,fleet_vehicle_tag_access_right,fleet.model_fleet_vehicle_tag,base.group_user,1,0,0,0
+hr_fleet_vehicle_state_access_right_user,fleet_vehicle_state_access_right,fleet.model_fleet_vehicle_state,base.group_user,1,0,0,0
+hr_fleet_vehicle_model_brand_access_right_user,fleet_vehicle_model_brand_access_right,fleet.model_fleet_vehicle_model_brand,base.group_user,1,0,0,0
+hr_fleet_vehicle_model_brand_category_right_user,fleet_vehicle_model_category_access_right,fleet.model_fleet_vehicle_model_category,base.group_user,1,0,0,0
+hr_fleet_vehicle_access_right_user,fleet_vehicle_access_right,fleet.model_fleet_vehicle,base.group_user,1,0,0,0
```

Comment:
We cannot modify access rules/access restrictions on a stable release when there's not critical bug to fix. 

It doesn't respect our stable policy. 

This is more a functional improvement than a bug to fix. So what I suggest:
- STABLE: Hide the stat button if you don't have access to the result.
- MASTER: Add the read access right to model_fleet_vehicle_assignation_log only, with a restriction to only see yours. And make the stat button works. 

Note:
Concerning those rules:
```xml
        <record id="hr_fleet_rule_log_services_visibility_base_user" model="ir.rule">
            <field name="name">User are able to see only their log services</field>
            <field name="model_id" ref="model_fleet_vehicle_log_services"/>
            <field name="groups" eval="[(4, ref('base.group_user'))]"/>
            <field name="domain_force">[('vehicle_id.driver_employee_id','in',user.employee_ids.ids)]</field>
        </record>
```
Did you check if the manager can actually see other records that his own ? I don't think that it is the case without a ``domain_force=[(1, '=', 1)]``

Reply:
I think there is a little misunderstanding here, that smartbutton shouldn't open you fleet.vehicles, i think geoffrey wants that to button to open fleet.vehicle.assignation.log records of the employee, so all those additional rules to access odometers and contracts should be removed